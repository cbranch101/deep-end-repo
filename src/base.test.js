test('A basic test', () => {
    expect(1).toMatchSnapshot('a snapshot')
})
