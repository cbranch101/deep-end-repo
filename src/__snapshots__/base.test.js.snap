// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A basic test: a snapshot 1`] = `1`;
